routes/api_v2_0.py

```diff
@@ -487,11 +487,13 @@ def subset_query(
     ),
     time: str = Query(description="", examples=["2283984000,2283984000"]),
     should_zip: str = Query("1", description="", examples=["1"]),
+    depth: str = Query(None, description="Optional depth index (e.g. 0 or 'bottom')"),
```

Comment:
```suggestion
    depth: str = Query(None, description="Optional depth index (e.g. 0 or 'bottom')"),
    should_zip: str = Query("1", description="", examples=["1"]),
```

This is a little nitpicky but we should group similar query components together so that users don't miss anything. I'd switch these two so that all of the data parameters (area, time, depth) are grouped. `output_format` would ideally be moved as well but this could break any existing API scripts.